docker-compose/standalone-d2cc/docker-compose.yml

```diff
@@ -261,6 +261,21 @@ services:
     networks:
       - gridcapa
 
+  cse-idcc-gridcapa-export:
```

Comment:
named idcc ! And you can still put it in 'acceptance-test' deployment :)

Reply:
Right!